docker/common/install_nixl.sh

```
@@ -18,14 +19,14 @@ pip3 install --no-cache-dir meson ninja pybind11
 git clone --depth 1 -b ${NIXL_VERSION} ${NIXL_REPO}
 cd nixl
 
-cuda_path=$(find / -name "libcuda.so.1" 2>/dev/null | head -n1)
-if [[ -z "$cuda_path" ]]; then
+CUDA_SO_PATH=$(find "/usr/local" -path "*/cuda*/compat/*/libcuda.so.1" 2>/dev/null | head -n1| xargs dirname)
+
+if [[ -z "$CUDA_SO_PATH/libcuda.so.1" ]]; then
```

Shellcheck (0.10.0)

[error] Argument to -z is always false due to literal strings.

(SC2157)